test/test_retryable_writes.py

```diff
@@ -469,6 +471,78 @@ def test_batch_splitting_retry_fails(self):
             self.assertEqual(final_txn, expected_txn)
         self.assertEqual(coll.find_one(projection={"_id": True}), {"_id": 1})
 
+    @client_context.require_multiple_mongoses
+    @client_context.require_failCommand_fail_point
+    def test_retryable_writes_in_sharded_cluster_multiple_available(self):
+        fail_command = {
+            "configureFailPoint": "failCommand",
+            "mode": {"times": 1},
+            "data": {
+                "failCommands": ["insert"],
+                "closeConnection": True,
+                "appName": "retryableWriteTest",
+            },
+        }
+
+        for mongos in client_context.mongos_seeds().split(","):
```

Comment:
`for mongos in client_context.mongoses:`

Reply:
That change causes the following error: 

```
h = ('localhost', 27017)

    def _connection_string(h):
>       if h.startswith(("mongodb://", "mongodb+srv://")):
E       AttributeError: 'tuple' object has no attribute 'startswith'

test/utils.py:544: AttributeError
```